tendrl/ceph_integration/manager/__init__.py

```diff
@@ -39,7 +37,13 @@ def main():
     complete = gevent.event.Event()
 
     def shutdown():
-        LOG.info("Signal handler: stopping")
+        try:
```

Comment:
abstract out this try/except block. This seems to have been copied in the entire PR, please abstract it